syntax = "proto3";

package std_daq_protocol;

message RequestNextImage {
    uint64 image_id = 1;
    bool new_request = 2;
}

message NextImageResponse {
    oneof action {
        NoImages no_image = 1;
        Ack ack = 2;
    }
}

message NoImages {
}

message Ack {
    uint64 image_id = 1;
}
